test/stubs.py

```diff
@@ -129,7 +129,7 @@ def stubInitConfig():
 
 # create a global CFG object,then replace it with our own that candlepin
 # read from a stringio
-#config.get_config_parser(config_file="test/rhsm.conf")
+#config.initConfig(config_file="test/rhsm.conf")
```

Comment:
Any reason to modify this commented code? Again could be an artefact of the cherry pick but it doesn't seem to be necessary afaict.

Reply:
it was one of the conflicts when I cherry-picked.  I mistakenly chose to go with what was in 1.24.  Will Change Back.